### PR TITLE
Update and clarify truffleruby docker README

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

File: truffleruby/README.md [hunk 1]
```diff
@@ -7,28 +7,38 @@ TruffleRuby is the [GraalVM](http://graalvm.org/) high-performance implementatio
 
 See https://github.com/graalvm/container/pkgs/container/truffleruby
 
-## How to use these images
+## Supported tags
 
-The images are intended for use in the **FROM** field of a downstream Dockerfile. For example, specify `FROM ghcr.io/graalvm/truffleruby:latest` or a version tag.
+* Oracle Linux 8: [`latest`/`VERSION`](https://github.com/graalvm/container/blob/master/truffleruby/Dockerfile.ol8).
```
Comment (ezzarghili) [inline truffleruby/README.md:12]: This is also Oracle Linux 8 slim, the difference is in the llvm-toolchain availability.

File: truffleruby/README.md [hunk 2]
```diff
@@ -7,28 +7,38 @@ TruffleRuby is the [GraalVM](http://graalvm.org/) high-performance implementatio
...
+## Supported tags
 
-The images are intended for use in the **FROM** field of a downstream Dockerfile. For example, specify `FROM ghcr.io/graalvm/truffleruby:latest` or a version tag.
+* Oracle Linux 8: [`latest`/`VERSION`](https://github.com/graalvm/container/blob/master/truffleruby/Dockerfile.ol8).  
 
-## Installing system packages
+* Oracle Linux 8 slim: [`slim`/`slim-VERSION`](https://github.com/graalvm/container/blob/master/truffleruby/Dockerfile.ol8-slim).  
 
-Use `microdnf install some-package`.
+* Oracle Linux 7: [`ol7`/`ol7-VERSION`](https://github.com/graalvm/container/blob/master/truffleruby/Dockerfile).
```
Comment (ezzarghili) [inline truffleruby/README.md:16]: Same as ol8 slim comment.

**Review by ezzarghili — CHANGES_REQUESTED** [review]

File: truffleruby/README.md (hunk 1 above)
Comment (eregon) [reply truffleruby/README.md:12]: Right, will clarify

Comment (eregon) [reply truffleruby/README.md:12]: done

**Comment by eregon** [comment]
Could you merge this?

There is still the issue that the post-install hook was not run for the slim images, but that seems orthogonal.

**Comment by eregon** [comment]
I filed a separate issue regarding the post-install hook for slim images: https://github.com/graalvm/container/issues/31

**Review by ezzarghili — APPROVED** [review]

**Pull request merged**